pkgs/by-name/fo/forgejo/generic.nix

```
@@ -83,6 +84,14 @@ buildGoModule rec {
 
   patches = [
     ./static-root-path.patch
+  ]
+  ++ lib.optionals lts [
+    (fetchpatch2 {
```

```suggestion
    (fetchpatch {
```
shouldn't this be fetchpatch because the patch contains short commit hashes?